app/views/layouts/tylium/_sidebar.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="main-sidebar sidebar-collapsed" data-behavior="main-sidebar sidebar-collapsed">
```

Comment:
TIL. I didn't realize you could add multiple values to a data behaviour. I assumed it would create a single behaviour named the full string "main-sidebar sidebar-collapsed".

Reply:
@brianp yup, we can stack them in the erb's and then use `~=` in JS when selecting elements by data-attribute.